src/components/SearchBar.jsx

```diff
@@ -0,0 +1,15 @@
+const SearchBar = ({ topics }) => {
+    return <section id="searchbar-container">
+        <form id="searchbar">
+            <label htmlFor="topic-filter-dropdown">Filter by Topic: </label>
+            <select name="topic" id="topic-filter-dropdown">
+                {topics.map(topic => {
+                    return <option key={topic.slug} value={topic.slug}>{topic.slug}</option>
+                })}
+            </select>
+            <button>Search</button>
+        </form>
+    </section>
```

Comment:
Though this is good to add, I would say that this element (though it isn't working yet) isn't in the scope of this ticket/task (it is in the scope of some tickets further on) - and it's important in PRs for the code just to meet the requirements of that ticket. Going forward try to keep this in mind 🧠